share/docker/Dockerfile

```diff
@@ -62,15 +64,23 @@ RUN apt-get -qqy update \
                                                  python3-setuptools \
                                                  python3-snappy \
                                                  python3-wheel \
-                                                 rake \
-                                                 ruby \
-                                                 ruby-dev \
                                                  source-highlight \
                                                  subversion \
                                                  valgrind \
  && apt-get -qqy clean \
  && rm -rf /var/lib/apt/lists
 
+# Install rbenv and ruby-build
```

Comment:
Please keep the ruby stuff all together in the Dockerfile. Consider moving this stuff down to line 174 or moving that stuff up to be part of this.

Reply:
Moved the install of rbenv down